### ISSUE TITLE
Review Red Hat OpenShift on IBM Cloud User access permissions

### ISSUE THREAD

**Issue by rnapoles-rh** [openshift/odo#4920]
/kind user-story

## User Story
As a IBM Cloud admin I want to understand how Red Hat OpenShift on IBM Cloud User access permissions work So that I can setup service ID accounts and api-keys for our test automation framework
## Acceptance Criteria 
- [ ] should be able to create service IDs with admin and developer access levels
- [ ] create api-tokens that do not expire
- [ ] be able to login with service ID and api-tokens 

## Links 
- Related Epic (mandatory): 

/kind user-story

**Comment by rnapoles-rh** [comment]
I tried running the scripts/configure-cluster/common/auth.sh script and it failed to create the accounts defined in the test-htpasswd file due to the IBM Cloud login dependency.
Reviewing https://cloud.ibm.com/docs/openshift?topic=openshift-access_reference

**Comment by rnapoles-rh** [comment]
Created service accounts for admin and developer and added permissions as required for each account, created api-keys for each account. Logged in as each account, downloaded kubeconfig files, executed oc commands to get information from cluster, however I was not able to create resources at developer or admin level got the following error:

**Comment by rnapoles-rh** [comment]
When logging in as a real user (non service ID) user is able to create resources. Will create IBMIDs for admin an developer and provide the requried access for each.  Then try running the teats for each user level.

**Comment by rnapoles-rh** [comment]
ododevauto is able to login and run tests as expected.

**Comment by dharmit** [comment]
@rnapoles-rh is this issue good to close or are we msising anything? What about the CI? Are we able to use this cluster on IBM Cloud for running our tests? 

Also token expiration issue cc @prietyc123

**Comment by rnapoles-rh** [comment]
The service account is now working as expected using API keys to login to the IBM Cloud and the ocp cluster in the IBM Cloud.

**Comment by dharmit** [comment]
Did you try to run the full test suite? How did that go?

**Comment by rnapoles-rh** [comment]
> Did you try to run the full test suite? How did that go?

Yes, the test results are listed at the end of issue #4798 some tests are failing, I am reviewing those failures.
